.babelrc

```diff
@@ -1,24 +1,47 @@
 {
   "only": "*.js",
   "env": {
-    "pre-node5": {
-      "presets": ["es2015-node4"],
+    "node4": {
       "plugins": [
-        ["array-includes"],
-        ["transform-inline-imports-commonjs"],
-        ["transform-runtime", { "polyfill": true, "regenerator": false }]
-      ]
+        ["transform-flow-strip-types"],
+        ["transform-object-rest-spread"],
+        ["transform-class-properties"],
+        ["transform-inline-imports-commonjs"]
+      ],
+      "presets": [["env", {
+        "targets": {
+          "node": 4
+        },
+        "useBuiltIns": true
+      }]]
+    },
+    "node6": {
```

Comment:
to what @volune said, maybe this can be "development", which babel uses as default.